library/src/main/res/values-sw600dp/dimens.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <dimen name="my_location_button_dimen">70dp</dimen>
+    <dimen name="hamburger_margin">48dp</dimen>
```

Comment:
`hamburger_margin` should not be in the **library** module

Reply:
Removed.